tests/system/action/motion_category/old_tests.py

```diff
@@ -0,0 +1,493 @@
+from unittest import TestCase
```

Comment:
Please remove this file if you have ported all tests.

Reply:
Removed.